Zend/zend_operators.c

```diff
@@ -30,6 +30,8 @@
 #include "zend_strtod.h"
 #include "zend_exceptions.h"
 #include "zend_closures.h"
+#include "zend_interfaces.h"
+#include "ext/spl/spl_iterators.h"
```

Comment:
Okay, I'll do that tomorrow. That's when PECL exts with a lot of helper funcs sometime get merged in (like SPL)...